Core/Frameworks/Baikal/Model/Config.php

```diff
@@ -174,21 +170,19 @@ function persist() {
                     }
 
                     break;
-                }
-                case "litteral": {
+
+                case "litteral":
                     $sValue = trim($sValue);
                     break;
-                }
-                default: {
+
+                default:
                     $sValue = "''";
                     break;
-                }
             }
 
             $mFound = false;
-            for ($k = ($iLines - 1); $k >= 0; $k--) {
+            for ($k = ($iLines - 1); $k >= 0; --$k) {
```

Comment:
It likes the `--$x` form in preference to `$x--` (pre rather than post increment, when it is OK). I could look for the fixer to disable, but there are not many lines of diff, so I left this "fixed".